app/controllers/organisations_controller.rb

```diff
@@ -69,10 +69,12 @@ def create
        redirect_to organisations_path and return false
      end
     @organisation = Organisation.new(org_params)
+    @categories_start_with = Category.first_category_name_in_each_type
```

Comment:
Yes, the change is in create in order to mirror the way update is written. Thus editing and creating a new organisation work correctly, with the only quirk that changes made into categories don't persist when there is a validation error.